mod-users/src/main/resources/karate-config.js

```diff
@@ -0,0 +1,87 @@
+function fn() {
```

Comment:
Is this a copy and paste of the config from another test project?

Reply:
Yes config file is same for all projects except 1 method which i have added for generating unique barcode.

**random_numbers: function() {
      var text = Math.floor(Math.random() * 10000);
      return text;
    }**